hb_report/constants.py

```diff
@@ -74,12 +74,11 @@
 start_stop Configuration.validated..Starting.heartbeat|Corosync.Cluster.Engine|Executive.Service.RELEASE|Requesting.shutdown|Shutdown.complete
 """
 
-PACKAGES = """pacemaker libpacemaker3
-pacemaker-pygui pacemaker-pymgmt pymgmt-client
-openais libopenais2 libopenais3 corosync libcorosync4
+PACKAGES = """pacemaker libpacemaker3 pacemaker-cli pacemaker-remote
+corosync libcorosync4
 resource-agents cluster-glue libglue2 ldirectord libqb0
-heartbeat heartbeat-common heartbeat-resources libheartbeat2
```

Comment:
Please don't remove these: For debugging purposes it would be interesting to include information about these packages if they are installed even though they should not be.

Reply:
That applies for all old packages: Please don't remove them from the list!